### PR TITLE
Add test for seed with foreign key

### PR DESCRIPTION
**Pull request by fujiwara402**

### CODE REVIEW

File: seed_test.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+		DBName: "",
+	}
+	db, err := sql.Open("mysql", sqlconf.FormatDSN())
+	if err != nil {
+		t.Fatal("Failed to connect to database:", err)
+	}
+	defer db.Close()
+
+	// create new database
+	query := fmt.Sprintf("CREATE DATABASE IF NOT EXISTS test")
```
Comment (meta-closure) [inline seed_test.go]: No need to use Sprintf function. simply define variable 
`query := "CREATE DATABASE IF NOT EXISTS test"`

File: seed_test.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+	if _, err := db.Exec(query); err != nil {
+		t.Fatal("Failed to create database if not exist:", err)
+	}
+	defer func() {
+		query = fmt.Sprint("DROP DATABASE test")
```
Comment (meta-closure) [inline seed_test.go]: `query = "DROP DATABASE test"`

File: seed_test.go [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+		if _, err := db.Exec(query); err != nil {
+			t.Fatal("Failed to drop test database:", err)
+		}
+	}()
+	query = fmt.Sprint("USE test")
```
Comment (meta-closure) [inline seed_test.go]: select declare `USE test` or use test prefix for each SQL

File: seed_test.go [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+	if _, err := db.Exec(query); err != nil {
+		t.Fatal("Failed to use test database:", err)
+	}
+
+	// create tables
```
Comment (meta-closure) [inline seed_test.go]: use query list and commit same time

```
qs := []string{query1, query2}
qs = append(qs, query3)
for _, q := range qs {
  db.Exec(q)
}
```

**Comment by fujiwara402** [comment]
@meta-closure Fixed!

**Pull request merged**